src/docs/content.go

```diff
@@ -616,4 +616,29 @@ Enterprise Performance Computing (EPC)`
 	for all those blocks.`
 	VerifyExample string = `
   $ singularity verify container.sif`
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	// help
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	HelpUse   string = `help <image path>`
```

Comment:
This should probably be:
```
HelpUse   string = `help <command|image path>`
```
Which would also change the other text so it isn't so focused on the help of an *image* but help in general

Reply:
good catch, thanks!